internal/store/leaf/recipients.go

```diff
@@ -279,6 +280,9 @@ func (s *Store) OurKeyID(ctx context.Context) string {
 		return kl[0]
 	}
 
+	debug.Log("WARNING: no owner key found in %v", recp)
+	out.Warning(ctx, "No owner key found. Make sure your key is trusted by GPG")
```

Comment:
Thanks for adding this, but you are mentioning GPG outside of the GPG specific code. This message would appear for all storage drivers.

Reply:
Ack. Thank you. I have reworded this a bit, but so far haven't found a better place to put it without doing some ugly changes that I don't want to do for this (the risk of breaking stuff here is just too high).

Reply:
I see. I guess the current commit is good then.